### PR TITLE
 A fix related to the change in local x <-> pixel row mapping

### PR DESCRIPTION
**Pull request by iouribelikov**

### CODE REVIEW

File: Detectors/ITSMFT/common/simulation/src/AlpideSimResponse.cxx [hunk 1]
```diff
@@ -204,28 +207,32 @@ bool AlpideSimResponse::getResponse(float vRow, float vCol, float vDepth, Alpide
...
   if (vCol < 0) {
     vCol = -vCol;
     flipCol = true;
   }
-  if (vCol > mColMax) return false;
+  if (vCol > mColMax)
+    return false;
   if (vRow < 0) {
     vRow = -vRow;
     flipRow = true;
```
Comment (shahor02) [inline Detectors/ITSMFT/common/simulation/src/AlpideSimResponse.cxx]: flipRow = false;

File: Detectors/ITSMFT/common/simulation/src/AlpideSimResponse.cxx [hunk 2]
```diff
@@ -204,28 +207,32 @@ bool AlpideSimResponse::getResponse(float vRow, float vCol, float vDepth, Alpide
     LOG(FATAL) << "response object is not initialized" << FairLogger::endl;
   }
   bool flipCol = false, flipRow = false;
```
Comment (shahor02) [inline Detectors/ITSMFT/common/simulation/src/AlpideSimResponse.cxx]: flipRow = true;

**Comment by iouribelikov** [comment]
@shahor02 You are right, Ruben.  Done, and squashed with the previous commit.

File: Detectors/ITSMFT/common/simulation/src/AlpideSimResponse.cxx [hunk 3]
```diff
@@ -24,9 +24,9 @@ using namespace o2::ITSMFT;
 using namespace std;
 
 ClassImp(o2::ITSMFT::AlpideSimResponse)
-ClassImp(o2::ITSMFT::AlpideRespSimMat)
+  ClassImp(o2::ITSMFT::AlpideRespSimMat)
```
Comment (sawenzel) [inline Detectors/ITSMFT/common/simulation/src/AlpideSimResponse.cxx]: this looks weird. Could you add a `;` at the end and reformat, please? Can be merged directly thereafter.

**Pull request merged**